docs/releases/0.3-NOTES.md

```diff
@@ -0,0 +1,22 @@
+## Release notes for poseidon v0.3
+
+# Significant changes
+Pod affinity and Pod Anti-Affinity enhancement.
+
+# Required Actions
+None
+
+# Highlighted changes
+* [Pod affinity and Anti affinity changes](https://github.com/kubernetes-sigs/poseidon/pull/118)
+
+# Full change list
+
+* [add some file to gitignore](https://github.com/kubernetes-sigs/poseidon/pull/102)
```

Comment:
I assume it's a minor change and no need to mention in release note.

Reply:
Done Thanks